test/mock-debug-test.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference path='../typings/chai/chai.d.ts'/>
+/// <reference path="../typings/mocha/mocha.d.ts"/>
+
+'use strict';
+
+import chai = require('chai');
+import { makeMockLogger, DebugLogger } from '../mock-debug';
```

Comment:
Would it make sense to give this the same API as the "debug" module?  It could be a drop-in replacement.